.github/workflows/python.yml

```diff
@@ -94,6 +94,7 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install pytest-cov
           python -m pip install -r plugins/policy-recommendation/requirements.txt
+          python -m pip install -r plugins/anomaly-detection/requirements.txt
```

Comment:
Please add anomaly-detection folder in other checks in this file, including `check-python-changes`, `python-lint`, `test-unit` etc. This will enable our CI job to check code style and run UT for the pyspark files.

Reply:
ack